model/src/main/kotlin/licenses/LicenseInfoResolver.kt

```diff
@@ -25,6 +25,7 @@ import org.ossreviewtoolkit.model.LicenseSource
 import org.ossreviewtoolkit.model.Provenance
 import org.ossreviewtoolkit.model.config.CopyrightGarbage
 import org.ossreviewtoolkit.model.config.PackageConfiguration
```

Comment:
commit: Regarding 'only the first of the curations is recorded in the resolved
license', as soon as the API is used this would be a logical change. As the plan is to migrate step wise this will lead to inconsistencies. What's the reason for not applying all curation and then later on fix that?

This logical change will silently lead to wrong concluded licenses, as our configuration assumes the previous behavior.

Reply:
The full statement is "if multiple curations produce the same result, only the first of the curations is recorded in the resolved license." This does not mean that no all curations are applied, but that if the application of multiple curations produces identical license findings, only one of the curations will be recorded in `ResolvedLicenseLocation.appliedCuration`.
For example, if there are license findings for `Apache-2.0` and `MIT` in the same file and lines, and both get separately curated to `BSD-3-Clause`, the resulting license finding would be identical. In this case only one of the curations would be recorded in the model.
AFAIK there is currently no code working with the applied curations, only with the resulting license findings, so I don't think this is a logical change.

Reply:
You're right, what you explained is an important aspect I missed completely.